packages/styles/scss/components/tile/_tile.scss

```diff
@@ -175,8 +177,8 @@
     right: 0;
     bottom: 0;
     display: flex;
-    width: $spacing-09;
-    height: $spacing-09;
+    width: calc(#{layout.density('padding-inline')} * 2 + 1rem);
+    height: calc(#{layout.density('padding-inline')} * 2 + 1rem);
```

Comment:
was trying to think if there is a better way to share this formula - it's seems a bit confusing and we're using it a lot - but I couldn't think of a better way 🤷🏽‍♀️

Reply:
Good point. I just pushed an update where I made a local variable (`$-icon-container-size`) out of this calculation and commented on the logic behind it. Do you think this makes sense?